src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,78 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Comparator;
+
 public class WorkWithFile {
+    private static String ERROR_READER = "Can't read file";
+    private static String WORD_DELI = ",";
+    private static String WORD_BUY = "buy";
+    private static String WORD_SUPPLY = "supply";
+    private static String WORD_RESULT = "result,";
+    private static String ERROR_WRITE = "Can't write data to file";
+    private static String NEW_LINE = System.lineSeparator();
 
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
![image](https://user-images.githubusercontent.com/81781479/195994829-0746bc7a-43f5-445b-878a-b2a4dc40fe4c.png)
lets divide your code into 3 methods
```suggestion
    public void getStatistic(String fromFileName, String toFileName) {
       String data = readFromFile(fromFileName);
       String report = getReport(data);
       writeToFile(report, toFileName);
```